README.md

```diff
@@ -169,24 +169,37 @@ If you intend to use [Stylelint support](#roadmap) (coming soon!), also note tha
 #### [6] SVG Store
 
 `src/assets/svg`
-  
+
 If you want to use the [SVG Store technique](https://css-tricks.com/svg-sprites-use-better-icon-fonts/), we added its support out of the box with the help of [webpack-svgstore-plugin](https://github.com/mrsum/webpack-svgstore-plugin).
 
-Here are the steps necessary to use it: 
+Here are the steps necessary to use it:
 1. Place all the necessary Svg files inside the `svg` folder
 2. Somewhere in your JS application, you need to create this variable and assignment:
     ```
     var __svg__ = { path: '../svg/**/*.svg', name: 'logos.svg' };
-    ```    
+    ```
     This will tell the SVG Store plugin [that it needs to generate the sprite file](https://github.com/mrsum/webpack-svgstore-plugin#2-put-function-mark-at-your-chunk).
-    
+
 **Note** that the plugin will add a `icon-` prefix to your file name as the `id` of the symbol in the sprite. There is no way to change this at the moment. Given that this is less than an ideal integration, we will look for better ways to generate the store [in the future](#roadmap).
 
 Also **note** that, if you want to reference the sprite file in your liquid templates, [you will need to make sure that the url is not parsed by Webpack](#how-to-prevent-webpack-from-parsing-some-liquid-methods-and-filters). You can do so by wrapping the liquid curly brackets in a single quote and the name of the sprite in double quotes, like so:
 ```
-<div data-some-attribute='{{ "logos.svg" | asset_url }}'></div>
+<div data-svgloader='{{ "logos.svg" | asset_url }}'></div>
 ```
 
+The sprite loader that comes with [webpack-svgstore-plugin]((https://github.com/mrsum/webpack-svgstore-plugin) won't work out of the box, instead given that you have included the above liquid snippet in your theme, this loader will suffice:
```

Comment:
You have an extra opening parenthesis in the `webpack-svgstore-plugin` link.

I think we should advocate for writing (and importing) a module instead of putting code directly in a liquid template.